exampleService/native/src/impl/ExampleService.cpp

```diff
@@ -12,24 +12,27 @@ namespace example {
 
       ::std::shared_ptr<ExampleService> ExampleService::S_INSTANCE = NULL;
 
-      ::std::shared_ptr<ExampleService> ExampleService::getInstance() {
+      ::std::shared_ptr<ExampleService> ExampleService::GetInstance() {
          if (S_INSTANCE == NULL) {
             S_INSTANCE = ndk::SharedRefBase::make<ExampleService>();
          }
          return S_INSTANCE;
       }
 
       ExampleService::ExampleService() {
-         readColumnFromCSV(0, cpuColumnData);
-         readColumnFromCSV(1, gpuColumnData);
-         readColumnFromCSV(2, ambientColumnData);
+         ReadColumnFromCSV(0, cpuColumnData);
```

Comment:
Folosirea de stil diferit pentru member variables, local variables, global variables creste capacitatea codului de a fi citit si inteles usor. In cazul de mai sus, imi este greu sa inteleg ce fel de variabla este `cpuColumnData` atunci cand folosesti acelasi stil de numire pentru mai multe tipuri de variable. Google coding style propune o varianta de stil: https://google.github.io/styleguide/cppguide.html#Variable_Names, exista si altele, ce este important e consistenta.

Reply:
Folosirea de "un-named constants" reduce "code readability". Un cititor trebuie sa gasesca definitia functiei "ReadColumnFromCSV" pentru a intelege ce reprezinta constanta "0", sau "1", etc..
Sugestia mea este sa folosesti "named constants" unde acest lucru creste "readability-ul".
e.g.
```suggestion
         const int kCpuColumnDataIndex = 0;
         ReadColumnFromCSV(kCpuColumnDataIndex, cpuColumnData);
```